VERSION

```diff
@@ -1 +1 @@
-0.8.0
+0.9.0
```

Comment:
Last change needed -- just revert the edits in this file.  It will also be auto-incremented based on your CHANGELOG entries.

Great work @arosen93!